x-pack/plugins/case/common/api/cases/case.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+
+import { CommentResponseRt } from './comment';
+import { UserRT } from '../user';
+
+const CaseBasicRt = rt.type({
+  description: rt.string,
+  state: rt.union([rt.literal('open'), rt.literal('closed')]),
```

Comment:
were we going to change this to `status`?

Reply:
I was thinking after this PR, in follow up. Since this pr is already big